src/modules/category/category.service.ts

```diff
@@ -54,7 +56,14 @@ export class CategoryService {
     try {
       return await this.prisma.category.delete({ where: { id } });
     } catch (e) {
-      throw new NotFoundException();
+      if (
+        e instanceof Prisma.PrismaClientKnownRequestError &&
+        e.code === 'P2025'
```

Comment:
is it any constant for it? 
or maybe prisma exports some types?

Reply:
I didn't found constants or enums for prisma error codes. I think we need create our constant, but I would be left it as string for now.